buildpacks-20/builder.toml

```diff
@@ -97,7 +97,7 @@ version = "0.16.1"
 [[order]]
   [[order.group]]
     id = "heroku/nodejs"
-    version = "0.6.5"
+    version = "0.6.7"
```

Comment:
what happened to 0.6.6?

Reply:
It was published as a CNB, but wasn't brought into the builder.